project/BuildExample.scala

```diff
@@ -229,4 +229,20 @@ object BuildExample extends sbt.Build {
         CrossVersion.binaryScalaVersion(scalaVersion.value)
     )
   ) dependsOn (streaming % "test->test; provided")
+
+  lazy val examples_redis = Project(
```

Comment:
please put examples into `experiments/redis` as well. We will split it into external and example modules when it gets mature.